modules/RTC/TPCUtils.js

```diff
@@ -354,12 +354,13 @@ export class TPCUtils {
      */
     replaceTrack(oldTrack, newTrack) {
         const mediaType = newTrack?.getType() ?? oldTrack?.getType();
+        const localTracks = this.pc.getLocalTracks(mediaType);
         const track = newTrack?.getTrack() ?? null;
         const isNewLocalSource = FeatureFlags.isMultiStreamSupportEnabled()
-            && this.pc.getLocalTracks(mediaType)?.length
+            && localTracks?.length
             && !oldTrack
             && newTrack
-            && !newTrack.conference;
+            && !localTracks.find(t => t === newTrack);
```

Comment:
why would `replaceTrack` allow to replace a track (as the replace target/ new track argument) that is already part of the peerconnection?

Reply:
This would happen in case of unmute operation when the track needs to be added back to the pc.

Reply:
oh okay, can you put a comment?